app/templates/core-data/admin-unit/index.hbs

```diff
@@ -40,32 +33,66 @@
                   {{/if}}
                 </:content>
               </Item>
-              <Item><:label>Regio : </:label>
+            {{/if}}
+
+            {{#if @model.isWorshipService}}
+              <Item>
+                <:label>Soort eredienst</:label>
+                <:content
+                >{{@model.adminUnit.recognizedWorshipType.label}}</:content>
```

Comment:
`recognizedWorshipType` and `denomination` are not defined neither in the frontend model for organization nor in the backend one, if you need this data you will need to add the property to both